### PR TITLE
fix swift dispersion  check

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**
It's related to a defect in backlog.
1. add insecure option for client self sign cert
2. add criticality option

and it depends on https://github.com/blueboxgroup/ursula-monitoring/pull/140

### CODE REVIEW

**Comment by nirajdp76** [comment]
Is the .sh being used anymore? What will happen on existing clouds where .sh check is already configured, will we end up with 2 checks for swift-disperssion?

File: roles/swift-common/defaults/main.yml [hunk 1]
```diff
@@ -123,6 +123,8 @@ swift:
         criticality: 'critical'
       fstab_mounts:
         criticality: 'critical'
+      check_swift_dispersion:
+        criticality: 'critical'
```
Comment (nirajdp76) [inline roles/swift-common/defaults/main.yml]: do we really need to run this check every 30 sec (i.e default)?

Comment (Xiaohua-Shen) [inline roles/swift-common/defaults/main.yml]: .sh will not be used anymore. only has 1 check for swift-dispersion, and related json file's content is changed (command is changed to .py ).
For the interval, let me change it to 2minutes.

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by Xiaohua-Shen** [comment]
retest

**Pull request merged**